### PR TITLE
Eject expo

### PR DESCRIPTION
**Pull request by ndeasy**
Migrating away from expo.

Updated Readme.

### CODE REVIEW

File: mobile/android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">Amputee Rehab</string>
```
Comment (adamjamesturner93) [inline mobile/android/app/src/main/res/values/strings.xml:2]: Can we rename these strings to Revenite?

File: mobile/app.json [hunk 2]
```diff
@@ -10,19 +10,19 @@
...
       "fallbackToCacheTimeout": 0
     },
     "assetBundlePatterns": [
       "**/*"
     ],
     "ios": {
-      "supportsTablet": true
-    }
+      "supportsTablet": true,
+      "bundleIdentifier": "org.AmputeeRehab"
```
Comment (adamjamesturner93) [inline mobile/app.json:21]: Rename to com.revenite (I won't comment on the name elsewhere though!)

File: README.md [hunk 3]
```diff
@@ -63,12 +63,52 @@ Install Expo CLI
 
 ```
  yarn global add expo-cli
+ yarn add expo
+```
+
+Install dependencies 
+```
+ cd mobile
+ npm install
```
[the remark this answers is not in the document]
Comment (klbennett) [reply README.md:73]: Updated

File: mobile/app.json (hunk 2 above)
Comment (klbennett) [reply mobile/app.json:21]: Name changed throughout

File: mobile/android/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (klbennett) [reply mobile/android/app/src/main/res/values/strings.xml:2]: Updated

**Pull request merged**